themes/half-life/half-life.theme.sh

```diff
@@ -16,7 +16,13 @@ function _omb_theme_PROMPT_COMMAND {
   local ps_path="${_omb_prompt_green}\w${_omb_prompt_normal}"
   local ps_user_mark="${_omb_prompt_red}λ${_omb_prompt_normal}"
 
-  PS1="$ps_username in $ps_path$(_omb_theme_half_way_prompt_scm) $ps_user_mark "
+  local python_venv
+  _omb_prompt_get_python_venv
+
+  PS1="$ps_username in $python_venv $ps_path$(_omb_theme_half_way_prompt_scm) $ps_user_mark "
```

Comment:
```suggestion
  PS1="$ps_username in $python_venv$ps_path$(_omb_theme_half_way_prompt_scm) $ps_user_mark "
```

Reply:
With `in $python_venv $ps_path`, there will be double space when no python virtual environment is activated.